SampleObjC/sdk-ios-objc/OCProfileViewController.m

```diff
@@ -0,0 +1,40 @@
+//
+//  ProfileViewController.m
+//  sdk-ios-objc
+//
+//  Created by Simon Hamadene on 16/10/2018.
+//  Copyright © 2018 Yoti Limited. All rights reserved.
+//
+
+#import "OCProfileViewController.h"
+
+@interface OCProfileViewController ()
+@property (weak, nonatomic) IBOutlet UIImageView *photoView;
+@property (weak, nonatomic) IBOutlet UILabel *phoneLabel;
+@property (weak, nonatomic) IBOutlet UILabel *givenNameView;
+@property (weak, nonatomic) IBOutlet UILabel *familyNameView;
+@property (weak, nonatomic) IBOutlet UILabel *genderView;
+@property (weak, nonatomic) IBOutlet UILabel *emailAddressView;
+@property (weak, nonatomic) IBOutlet UILabel *postalAddressView;
+@property (weak, nonatomic) IBOutlet UILabel *dateOfBirthView;
```

Comment:
Please make them `nonnull` to match the swift implementation

Reply:
They are weak they can't be non null.